### PR TITLE
Jp plugin

### PR DESCRIPTION
**Pull request by macanudo527**
Adding the JP Plugin to RP2.

### CODE REVIEW

File: src/rp2/plugin/country/jp.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+from rp2.abstract_country import AbstractCountry
+from rp2.rp2_main import rp2_main
+
+
+# JP-specific class
+class JP(AbstractCountry):
+    def __init__(self) -> None:
+        super().__init__("jp", "jpy")
+
+    # Measured in days
```
Comment (eprbell) [inline src/rp2/plugin/country/jp.py:25]: Maybe add a comment saying that JP has no long-term capital gains

File: src/rp2/plugin/country/jp.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    def long_term_capital_gain_period(self) -> int:
+        return sys.maxsize
```
Comment (eprbell) [inline src/rp2/plugin/country/jp.py:27]: add `import sys` at the beginning

File: src/rp2/plugin/country/jp.py [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+# JP-specific entry point
+def rp2_entry() -> None:
+    rp2_main(JP())
```
Comment (eprbell) [inline src/rp2/plugin/country/jp.py:32]: Add newline

**Review by eprbell — CHANGES_REQUESTED** [review]
Nice, first supported country after US!

**Comment by macanudo527** [comment]
These issues should be resolved now.

**Review by eprbell — APPROVED** [review]
LGTM!

**Comment by eprbell** [comment]
BTW, you need to add a JP country plugin to DaLI as well: https://github.com/eprbell/dali-rp2/blob/main/README.dev.md#country-plugin-development

**Pull request merged**